command/polybftmanifest/manifest_init.go

```diff
@@ -153,31 +153,30 @@ func (p *manifestInitParams) getValidatorAccounts() ([]*polybft.Validator, error
 		validators := make([]*polybft.Validator, len(p.validators))
 		for i, validator := range p.validators {
 			parts := strings.Split(validator, ":")
-
 			if len(parts) != 4 {
 				return nil, fmt.Errorf("expected 4 parts provided in the following format "+
-					"<nodeId:ECDSA address:blsKey:blsSignature>, but got %d part(s)",
+					"<P2P multi address:ECDSA address:public BLS key:BLS signature>, but got %d part(s)",
 					len(parts))
 			}
 
-			if len(parts[0]) != nodeIDLength {
-				return nil, fmt.Errorf("invalid node id: %s", parts[0])
+			if _, err := multiaddr.NewMultiaddr(parts[0]); err != nil {
+				return nil, fmt.Errorf("invalid P2P multi address '%s' provided: %w ", parts[0], err)
 			}
 
-			if len(parts[1]) != ecdsaAddressLength {
-				return nil, fmt.Errorf("invalid address: %s", parts[1])
+			if len(strings.TrimPrefix(parts[1], "0x")) != ecdsaAddressLength {
+				return nil, fmt.Errorf("invalid ECDSA address: %s", parts[1])
 			}
 
 			if len(strings.TrimPrefix(parts[2], "0x")) != blsKeyLength {
-				return nil, fmt.Errorf("invalid bls key: %s", parts[2])
+				return nil, fmt.Errorf("invalid BLS key: %s", parts[2])
 			}
 
 			if len(parts[3]) != blsSignatureLength {
-				return nil, fmt.Errorf("invalid bls signature: %s", parts[3])
+				return nil, fmt.Errorf("invalid BLS signature: %s", parts[3])
 			}
 
 			validators[i] = &polybft.Validator{
-				NodeID:       parts[0],
+				MultiAddr:    parts[0],
```

Comment:
Use `strings.TrimPrefix(parts[1], "0x")` and `strings.TrimPrefix(parts[2], "0x")` here also

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1272/commits/13cda702f09e49d9b7533bb7d11a7907c003faa6